packages/terra-list/src/terra-dev-site/test/list/list-item/DefaultListItem.test.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 
 import ListItem from '../../../../ListItem';
+import List from '../../../../List';
 
-export default () => <ListItem content={<p>test content</p>} style={{ listStyle: 'none' }} />;
```

Comment:
The point of initially reacting these `ListItem` examples as standalone without being wrapped in the `List `component was to ensure the `ListItem`s would render as expected without influence of the `List`. Given the ListItem should never be rendered outside of the `List` given the implementation expectations and accessibility issues that would occur, it almost seems as if these tests are not gaining us anything.

I would be curious on what others thoughts if removed ListItem visual-regression tests and example and only tested the ListItem via jest. My thought that this is okay is that each List example is composed of the ListItems, which provide they render as expected.

@mjhenkes @ryanthemanuel @bjankord @JakeLaCombe

Reply:
@emilyrohrbough your argument makes sense to me. Remove list item visual regression, test list item through jest and list item in a list with wdio.